src/components/homepage/HomeNavBar.js

```diff
@@ -1,85 +1,72 @@
 
-  const handleLogout = () => props.onLogout();
+  const [open, setOpen] = useState(false);
 
+  //toggle's App 'showNav' state value, and local useState value
+  const toggle = () => {
+    console.log("clicked");
+    props.toggleClassOnNavOpen();
+    setOpen(!open);
+  };
+  
   return (
-    <div className="header">
-        <div className="nav-logo">
-        <FlowerHeartLogoMobile className="logo-main"/>
+    <div className="nav-container">
+      <nav>
+        {/* using 'open' state value to conditionally mount/unmount menu icons */}
+        {open ? (
+          <a className="nav-btn-toggle" onClick={toggle}>
+            <MenuClose />
+          </a>
+        ) : (
+          <a className="nav-btn-toggle" onClick={toggle}>
+            <MenuOpen />
+          </a>
+        )}
+        <div className="left-menu">
+          <Link to="/dashboard">Dashboard</Link>
+          <Link to="/journal">Journal</Link>
+          <Link to="/feeling_tracker">Feelings Tracker</Link>
+          <Link to="/todos">Todos</Link>
         </div>
-    
-      <Navbar className="header-nav"
-      sticky="top"
-        collapseOnSelect
-        expand="lg"
-        style={{ backgroundColor: "white"}} 
-      >
-        {props.loggedIn ? (
-          <>
-            <Navbar.Toggle aria-controls="basic-navbar-nav" />
-            <Navbar.Collapse
-              className="justify-content-end"
-              id="basic-navbar-nav"
-            >
-              <Container className="text-center" >
-              <Nav>
-                {token ? (
-                  <Nav.Link as={Link} to="/dashboard">
-                    Dashboard
-                  </Nav.Link>
-                ) : null}
-
-                {token ? (
-                  <Nav.Link as={Link} to="/journal">
-                    Journal
-                  </Nav.Link>
-                ) : null}
-
-                {token ? (
-                  <Nav.Link as={Link} to="/feeling_tracker">
-                    Feelings Tracker
-                  </Nav.Link>
-                ) : null}
-
-                {token ? (
-                  <Nav.Link as={Link} to="/todos">
-                    To-Do List
-                  </Nav.Link>
-                ) : null}
 
-                {token ? (
-                  <Nav.Link onClick={handleLogout}>Sign Out</Nav.Link>
-                ) : (
-                  <Nav.Link as={Link} to="/login">
-                    Login
-                  </Nav.Link>
-                )}
-              </Nav>
-              </Container>
-            </Navbar.Collapse>
-          </>
-        ) : null}
-      </Navbar>
+        <Link className="logo" to="/dashboard">
+          <FlowerHeartLogoMobile />
+        </Link>
+      </nav>
+      {/* using 'showNav' to conditionally render overlay, animations only run on mount and unmount, 
+      so this is currently necessary  */}
+      {props.showNav ? <NavOverlay open={open}></NavOverlay> : null}
```

Comment:
This could be written 
```suggestion
      {props.showNav && <NavOverlay open={open} />}
```